src/main/external-resources/renderers/Samsung-4K-2019.conf

```diff
@@ -0,0 +1,104 @@
+Supported = f:mov             v:mp4|mpeg1|mpeg2|vc1|wmv|h263|vp6		a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/quicktime		w:1920	h:1080	b:160000000
+Supported = f:flv             v:h264						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-flv			w:4096	h:2160	b:320000000
+Supported = f:flv             v:mjpeg						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-flv			w:4096	h:2160	b:640000000
+Supported = f:flv             v:mp4|mpeg1|mpeg2|vc1|wmv|h263|vp6		a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-flv			w:1920	h:1080	b:160000000
+Supported = f:mpegps|mpegts   v:h264						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/mpeg			w:4096	h:2160	b:320000000
+Supported = f:mpegps|mpegts   v:mjpeg						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/mpeg			w:4096	h:2160	b:640000000
+Supported = f:mpegps|mpegts   v:mp4|mpeg1|mpeg2|vc1|wmv|h263|vp6		a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/mpeg			w:1920	h:1080	b:160000000
+Supported = f:mpegts          v:h265						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/vnd.dlna.mpeg-tts	w:7680	h:4320	b:800000000
+Supported = f:wmv             v:h264						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-wmv		w:4096	h:2160	b:320000000
+Supported = f:wmv             v:mjpeg						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-wmv		w:4096	h:2160	b:640000000
+Supported = f:wmv             v:mp4|mpeg1|mpeg2|vc1|wmv|h263|vp6		a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-wmv		w:1920	h:1080	b:160000000
+Supported = f:asf             v:h264						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-asf		w:4096	h:2160	b:320000000
+Supported = f:asf             v:mjpeg						a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-asf		w:4096	h:2160	b:640000000
+Supported = f:asf             v:mp4|mpeg1|mpeg2|vc1|wmv|h263|vp6		a:ac3|lpcm|adpcm|aac-lc|he-aac|wma|eac3|mp3		m:video/x-ms-asf		w:1920	h:1080	b:160000000
+Supported = f:webm            v:vp8						a:vorbis						m:video/webm			w:1920	h:1080	b:160000000
+Supported = f:webm            v:vp9						a:vorbis						m:video/webm			w:4096	h:2160	b:160000000
+
+# Supported audio formats:
+Supported = f:mp3   	m:audio/mpeg
+Supported = f:aac   	m:audio/x-m4a
+Supported = f:flac   	m:audio/x-flac		n:2
+Supported = f:vorbis   	m:audio/vorbis		n:2
+Supported = f:wma   	m:audio/x-ms-wma
+Supported = f:wav   	m:audio/wav
+Supported = f:ape   	m:audio/ape
+Supported = f:aiff   	m:audio/L16
+Supported = f:alac   	m:audio/m4a
+
+# Supported image formats:
+Supported = f:bmp    	m:image/bmp			w:4096	h:4096
+Supported = f:jpg    	m:image/jpeg		w:15360	h:8640
+Supported = f:png    	m:image/png			w:4096	h:4096
+Supported = f:mpo    	m:image/mpo			w:15360	h:8640
+
+# Supported subtitles formats:
+SupportedExternalSubtitlesFormats = SAMI,SUBRIP,MICRODVD,ASS,TEXT,TX3G
```

Comment:
@SubJunk if you approce the #961 than this could be implemented correctly for all containers.

Reply:
That's done now